Dockerfile

```diff
@@ -48,7 +49,33 @@ RUN apt-get update && \
   php7.2-xmlrpc \
   php7.2-zip
 
-RUN apt-get -y  remove ssmtp && \
+# Support php-rdkafka
+RUN apt-get -y install \
+    git \
+    libpthread-stubs0-dev \
+    php7.2-dev \
+  && git clone https://github.com/edenhill/librdkafka.git \
+  && cd librdkafka \
+  && git checkout v1.6.1 \
+  && ./configure \
+  && make -j 2 \
+  && make install \
+  && cd .. \
+  && git clone https://github.com/arnaud-lb/php-rdkafka.git \
+  && cd php-rdkafka \
+  && git checkout 5.0.0 \
+  && phpize \
+  && ./configure \
+  && make all -j 2 \
+  && make install \
+  && cd .. \
+  && rm -rf librdkafka php-rdkafka \
+  && apt-get remove -y php7.2-dev \
+  && echo "extension = rdkafka.so" > /etc/php/7.2/mods-available/rdkafka.ini \
+  && phpenmod rdkafka
+
+RUN apt-get -y remove ssmtp git && \
```

Comment:
removing git from here will not remove it from the previous layer, which will remain larger than necessary. and actually i just realized every RUN statement that runs `apt-get install` should also run `apt-get update` prior and then the various cleaning commands after the installation, otherwise there will be caching and size issues.